maestrowf/interfaces/script/fluxscriptadapter.py

```diff
@@ -151,22 +152,28 @@ def get_parallelize_command(self, procs, nodes=None, **kwargs):
         and procs.
         """
 
-        args = [
-            'env',
-            '-u', 'FLUX_JOB_ID',
-            '-u', 'PMI_FD',
-            '-u', 'PMI_RANK',
-            '-u', 'PMI_SIZE',
-            'mpirun',
-            '-gpu',
-            '-mca', 'plm', 'rsh',
-            # '-mca', 'plm_rsh_agent', 'rsh',
-            '--map-by', 'node']
-        args.extend(['-hostfile','$HOSTF'])
+        # args = [
+        #     'env',
+        #     '-u', 'FLUX_JOB_ID',
+        #     '-u', 'PMI_FD',
+        #     '-u', 'PMI_RANK',
+        #     '-u', 'PMI_SIZE',
+        #     'mpirun',
+        #     '-gpu',
+        #     '-mca', 'plm', 'rsh',
+        #     # '-mca', 'plm_rsh_agent', 'rsh',
+        #     '--map-by', 'node']
+        # args.extend(['-hostfile','$HOSTF'])
+        # args.extend([
+        #     '-n',
+        #     str(procs),
+        #     # '/usr/gapps/kras/install/bin/autobind', # bind cores and GPUs
+        #     ])
+        args = ['/usr/gapps/kras/install/bin/mpiexec.hydra']
```

Comment:
This looks platform specific -- we should probably assume mpiexec is somewhere in the path.